packages/common/src/items/removeResource.ts

```diff
@@ -0,0 +1,51 @@
+import { IUserRequestOptions } from "@esri/arcgis-rest-auth";
```

Comment:
Now that I'm looking at this function, I see that the only thing we really get out of wrapping the `removeItemResource` function from `arcgis-rest-portal` is that we can return our own error message if there's a failure. I guess I'm now reconsidering if we really need this wrapper function